packages/core/__tests__/interpreter/base.js

```diff
@@ -143,7 +143,7 @@ test('saga yielded falsy values', () => {
   const task = middleware.run(genFn)
   const expected = [false, undefined, null, '', 0, NaN]
   return task.toPromise().then(() => {
-    expect(isNaN(last(expected))).toBe(true) // saga must inject back yielded falsy values
+    expect(isNaN(last(actual))).toBe(true) // saga must inject back yielded falsy values
```

Comment:
would the previous version fail if a more correct `Number.isNaN` would be used here?
```suggestion
    expect(Number.isNaN(last(actual))).toBe(true) // saga must inject back yielded falsy values
```

Reply:
Tests pass all the same with your suggestion using `Number.isNaN(last(expected))`

Reply:
I'm not sure what I was even asking here 🤣 my question didn't make a lot of sense.